### ISSUE TITLE
How can I disable simplebar for responsiveness.?

### ISSUE THREAD

**Issue by ShekhSaifuddin007** [Grsmto/simplebar#524]
How can I disable or remove simplebar based on screen size in a vue template.  I want to disable this simplebar when screen goes to 768px width. How can I achieve this, help me somebody please

**Comment by gitcatrat** [comment]
I'd recommend to only check the screen size on page load and decide based on that if you want to use simplebar or not. Otherwise the breaking point can get quite janky and bad for user experience. Developers are really the only people in the world who play with window size and expect something to happen, user would find it weird that scrollbar changes if they narrow the viewport to <700px or something.

**Comment by Grsmto** [comment]
@kalnode you're right. Thing is SimpleBar changes the DOM structure and it can't bring back the initial structure once it's been instantiated.
I would advice to only enable it when needed (mobile first) with for example matchMedia:

```
const shouldActivateSimplebar = window.matchMedia("(min-width: 600px)")
let simplebarInstance;

if (shouldActivateSimplebar) {
   simplebarInstance = new SimpleBar(...)
}
```

then to "disable" it you could hide it in CSS the same way with media queries.
